workflows/raredisease.nf

```diff
@@ -70,7 +73,7 @@ include { MULTIQC } from '../modules/nf-core/modules/multiqc/main' addParams( op
 // SUBWORKFLOW: Consists entirely of nf-core/modules
 //
 
-include { PREPARE_GENOME } from '../subworkflows/local/prepare_genome' addParams( bwamem2_idx_options: publish_idx_options )
+include { PREPARE_GENOME } from '../subworkflows/local/prepare_genome' addParams( bwamem2_idx_options: modules['bwa_mem2_index'], samtools_faidx_options: modules['samtools_faidx'] )
```

Comment:
A line break after addParams would help with readability. Same as done for the MAPPING subworkflow